engine/src/main/java/org/terasology/engine/GameEngine.java

```diff
@@ -45,19 +62,14 @@
     void close();
 
     /**
-     * @return Whether the engine has been uninitialized
+     * @return Whether the engine is running - this is true from the point run() is called to the point shutdown is complete
      */
-    boolean isUninitialized();
-
-    /**
-     * @return Whether the engine has been initialized
-     */
-    boolean isInitialized();
```

Comment:
Please confirm: there is no "initializing" state because no party external to the engine would be interested in this phase, right?